### ISSUE TITLE
injection-tck uses same package as the javax.inject-tck

### ISSUE THREAD

**Issue by hgschmie** [eclipse-ee4j/injection-tck#18]
Not really a bug, more an annoyance. 

It is not possible to use both TCKs (e.g. to test an injection framework that supports both javax.inject and jakarta.inject) with both TCKs as they overlap in class and package names.

**Comment by Ladicek** [comment]
Workarounds should be pretty straightforward I guess? E.g. if I were to run the TCK with Maven, I would have 2 Maven modules or 2 Surefire executions, which would allow me to setup different classpaths.

**Comment by hgschmie** [comment]
agreed (which is why I consider this more an annoyance than a bug). In this specific case (guice), the Tck is one test of very many tests and adding a new module just for this one test execution seems to be cumbersome.
